### PR TITLE
Buildscan url

### LINKED ISSUE #85: GitHub Actions: Surface the Gradle build-scan
As described here 
https://goobar.io/surfacing-gradle-build-scans-within-github-actions/#how-to-enable-build-scans-for-your-android-project

Also the unit tests are currently not run.

We shall run those which are not in the `testing.failing` package

### PR DESCRIPTION
**Pull request by jmfayard**
Closes #85 GitHub Actions: Surface the Gradle build-scan

### CODE REVIEW

File: .github/workflows/runOnGitHub.yml [hunk 1]
```diff
@@ -21,9 +21,24 @@ jobs:
...
+      - id: gradle
+        uses: eskatos/gradle-command-action@v1
         with:
           arguments: runOnGitHub
           wrapper-cache-enabled: true
           dependencies-cache-enabled: true
           configuration-cache-enabled: true
+      - id: buildscan
+        name: Add Build Scan URLs to Pull Request
+        uses: mshick/add-pr-comment@v1
```
Comment (LouisCAD) [inline .github/workflows/runOnGitHub.yml:33]: Can we point to a non dynamic version for this third-party action?

**Comment by LouisCAD** [comment]
Damn yaml!

**Comment by jmfayard** [comment]
I've read the docs and from what I understand `GITHUB_TOKEN` will work only for us collaborators, not for contributors on their forks

https://github.com/mshick/add-pr-comment

That's not good. 

So I think we do need to put the token of `jmfayard-bot` in the secrets

**Comment by LouisCAD** [comment]
Secrets are never available for actions run by non collaborators AFAIK (for security reasons). I'll see if there's a way to allow this specific one though.

**Pull request merged**